modules/fb_instant_articles_display/fb_instant_articles_display.module

```diff
@@ -13,15 +13,15 @@
 */
 function fb_instant_articles_display_init() {
  // Configure Instant Articles Transformer logger to append to Drupal log if enabled
- $enable_transformer_logging = variable_get('fb_instant_articles_display_enable_transformer_logging', FALSE);
+ $enable_transformer_logging = variable_get('fb_instant_articles_display_enable_transformer_logging');
```

Comment:
@m4olivei you're spot on: those are exactly the changes we'll have to make in the transformer logger: using a renderer (I think those can be customized in `log4php`) to group those lines in a single messages that make sense whether read chronologically or reverse. Feel free to file the issue in the SDK repo and we'll get right to it post the F8 launch.